_sass/base.scss

```diff
@@ -0,0 +1,98 @@
+.site-header {
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100%;
+	border-top: none;
+	border-bottom-color: transparent;
+	background-color: $dark-background-color;
+
+	min-height: 70px;
+	transition: all .5s;
+
+	display: flex;
+	align-items: center;
+
+	&.site-header-scrolled {
+		min-height: 56px;
+	}
+
+	& > .wrapper {
+		width: 100%;
+		position: relative;
+	
+		.site-title {
+			letter-spacing: unset;
+			color: $text-color-light;
+			text-decoration: none;
+		}
+
+		.site-nav .trigger .page-link {
+			color: $text-color-light;
+			margin-left: 1em;
+			margin-right: 1em;
+			@include media-query($on-palm) {
+				color: $text-color;
```

Comment:
Why is this a specific override for mobile devices? What does this do?

Reply:
Moved it to the override below -- this is because the nav-links are in a hamburger menu on mobile, and so need to be a different color.